content/docs/iac/concepts/functions/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title_tag: "Functions"
+meta_desc: Learn about the three types of functions available in Pulumi programs - provider functions, get functions, and resource methods.
+title: Functions
+h1: Functions
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    iac:
+        name: Functions
+        parent: iac-concepts
+        weight: 60
+        identifier: iac-concepts-functions
+    concepts:
+        name: Functions
+        parent: Overview
+        weight: 6
+---
+
+Pulumi provides three types of functions that you can use in your programs to interact with cloud resources and retrieve data from cloud providers. Understanding these different function types and when to use them is essential for writing effective Pulumi programs.
+
+## Provider functions
+
+**[Provider functions](/docs/iac/concepts/functions/provider-functions/)** are functions exposed in Pulumi providers that allow you to query cloud APIs to retrieve data that is not part of a resource. These functions are often used to look up information about existing cloud resources that you need in your Pulumi program.
+
+For example, you might use a provider function to get the latest virtual machine image ID for a specific operating system.
+
+## Get functions
+
+**[Get functions](/docs/iac/concepts/functions/get-functions/)** are static functions available on all Pulumi resource types that allow you to reference an existing resource that is not managed by Pulumi. Unlike the `pulumi import` command which brings resources under Pulumi management, get functions simply allow you to read the properties of existing resources.
```

Comment:
The phrasing is a bit language specific. In Go, they are just free-standing functions.

Reply:
Y'know... I did struggle with this. I originally wanted to put the examples of each type in every language, but I could not get the dang invokes to fit on one line.

Added a note for the future: https://github.com/pulumi/docs/issues/15457